tests/test_prereleases_modular.py

```
@@ -261,9 +261,10 @@ def test_prerelease_history_build_simplified_history(prerelease_manager):
         },
     ]
 
-    # Test with expected version "2.7.14"
+    # Stable 2.7.13 admits prerelease bases strictly newer (2.7.14); the 2.7.13
+    # delete commit is rejected (not strictly newer than stable).
     result, seen_shas = prerelease_manager.build_simplified_prerelease_history(
```

Ruff (0.15.20)

[warning] Unpacked variable `seen_shas` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)